### PR TITLE
Caesar Cipher in Python

### PR DESCRIPTION
**Pull request by ghost**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description
I have implemented Caesar Cipher in PYTHON
To shed some light on cryptographic algorithms, we begin with coding the basic Caesar Cipher which encrypts and decrypts messages with a specific shift value

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#422

### CODE REVIEW

File: Python/cryptography/Caesar_Cipher.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+        """Saving current character in current_character"""
+        current_character = text[character]
+
+        """If current characater is space adding it directly to message"""
+        if current_character == ' ':
+            message += current_character
+            continue
+
+        """If current characater is an uppercase letter adding it after shifting with key value to message"""
```
Comment (atarax665) [inline Python/cryptography/Caesar_Cipher.py]: You can use # instead of """...."""  for single line comments.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/cryptography/Caesar_Cipher.py (hunk 1 above)
Comment (ghost) [reply Python/cryptography/Caesar_Cipher.py]: I have made them one line comments

**Comment by ghost** [comment]
@atarax665 Can I get some help, I am getting a merge conflict : "Merge Conflicts Labeler / labeler (pull_request) Failing after 55s — labeler"

**Comment by ghost** [comment]
@atarax665 @SKAUL05 Please Review the pr, I have resolved the conflicts and made the requested changes

**Review by atarax665 — APPROVED** [review]
LGTM :+1:

**Review by vatsalkesarwani12 — APPROVED** [review]
✅

**Comment by ghost** [comment]
@SKAUL05 Please review this PR

**Review by SKAUL05 — APPROVED** [review]

**Pull request merged**